src/cljs/athens/views/blocks/toggle.cljs

```diff
@@ -44,18 +45,18 @@
 
 
 (defn toggle-el
-  [{:block/keys [open uid children]} state linked-ref]
-  (if (seq children)
-    [:button (stylefy/use-style block-disclosure-toggle-style
-                                {:class    (if (or (and (true? linked-ref) (:linked-ref/open @state))
-                                                   (and (false? linked-ref) open))
-                                             "open"
-                                             "closed")
-                                 :tab-index 0
-                                 :on-click (fn [_]
+  [{:block/keys [open uid]} state linked-ref]
+  [:button (stylefy/use-style block-disclosure-toggle-style
+                              {:class    (if (or (and (true? linked-ref) (:linked-ref/open @state))
+                                                 (and (false? linked-ref) open))
+                                           "open"
+                                           "closed")
+                               :tab-index 0
+                               :on-click (fn [e]
+                                           (doall
+                                             (.. e stopPropagation)
                                              (if (true? linked-ref)
                                                (swap! state update :linked-ref/open not)
-                                               (toggle [:block/uid uid] open)))})
-     [:> KeyboardArrowDown {:style {:font-size "16px"}}]]
-    [:span (stylefy/use-style block-disclosure-toggle-style)]))
+                                               (toggle [:block/uid uid] open))))})
+   [:> KeyboardArrowDown {:style {:font-size "16px"}}]])
```

Comment:
This is an example of how we were leaving a placeholder element to make the styling work right. That styling logic is all handled by the container's grid settings now.